### PR TITLE
Added new controller for character details

### PR DESCRIPTION
**Pull request by NikolaBlagoevski**

### CODE REVIEW

**Comment by NikolaBlagoevski** [comment]
[pull request from CellViewController to main](https://trello.com/c/ICSPlKvB/57-pull-request-from-cellviewcontroller-to-main)

File: test1/CellForCharacter.swift [hunk 1]
```diff
@@ -27,6 +27,9 @@ class CellForCharacter: UICollectionViewCell{
         myLabel.text = "This is some text"
         //myLabel.backgroundColor = .green
         myLabel.textAlignment = .center
+        myLabel.lineBreakMode = .byWordWrapping
+        myLabel.numberOfLines = 0
+        //myLabel.adjustsFontSizeToFitWidth = true
```
Comment (martinpesevski) [inline test1/CellForCharacter.swift]: do not commit commented out code

File: test1/CellViewController.swift [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+import UIKit
+import SnapKit
+import SDWebImage
+
+class NikolaAwesomeLabel: UILabel{
+    override init(frame: CGRect){
+        super.init(frame: frame)
+        self.backgroundColor = .white
+        self.font = .systemFont(ofSize: 20)
+        self.textAlignment = .left
```
Comment (martinpesevski) [inline test1/CellViewController.swift]: left is default alignment, you don't need this

File: test1/CellViewController.swift [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+//
+
+import UIKit
+import SnapKit
+import SDWebImage
+
+class NikolaAwesomeLabel: UILabel{
+    override init(frame: CGRect){
+        super.init(frame: frame)
+        self.backgroundColor = .white
```
Comment (martinpesevski) [inline test1/CellViewController.swift]: this is default, you don't need.

File: test1/CellViewController.swift [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+class CellViewController: UIViewController{
+    
+
```
Comment (martinpesevski) [inline test1/CellViewController.swift:26]: try to not leave double empty lines

File: test1/CellViewController.swift [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        //view.addSubview(tableView)
+//      tableView.delegate = self
+//      tableView.dataSource = self
+        makeStackView()
+    }
+    
+    lazy var image: UIImageView = {
```
Comment (martinpesevski) [inline test1/CellViewController.swift:44]: property definitions go before viewdidload and any other functions

File: test1/CellViewController.swift [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+        let temp = UIImageView()
+        temp.sd_setImage(with: URL(string: person.image))
+        return temp
+    }()
+    
+    lazy var label1: NikolaAwesomeLabel = {
```
Comment (martinpesevski) [inline test1/CellViewController.swift]: we need better names than label1.2.3 etc

File: test1/CellViewController.swift [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+        let tmp = NikolaAwesomeLabel()
+        var txt = person.status
+        if txt.count < 1 {
```
Comment (martinpesevski) [inline test1/CellViewController.swift]: you can use txt.isempty

**Review by martinpesevski — CHANGES_REQUESTED** [review]

File: test1/CellForCharacter.swift (hunk 1 above)
Comment (NikolaBlagoevski) [reply test1/CellForCharacter.swift]: I removed all the commented out code

File: test1/CellViewController.swift (hunk 2 above)
Comment (NikolaBlagoevski) [reply test1/CellViewController.swift]: deleted that line

File: test1/CellViewController.swift (hunk 3 above)
Comment (NikolaBlagoevski) [reply test1/CellViewController.swift]: deleted self.backgroundColor = .white

File: test1/CellViewController.swift (hunk 4 above)
Comment (NikolaBlagoevski) [reply test1/CellViewController.swift:26]: removed all of those

File: test1/CellViewController.swift (hunk 5 above)
Comment (NikolaBlagoevski) [reply test1/CellViewController.swift:44]: I moved the makeStackView() function from viewDIdLoad() to init()

File: test1/CellViewController.swift (hunk 6 above)
Comment (NikolaBlagoevski) [reply test1/CellViewController.swift]: the names of the labels are changed and are easy to understand

File: test1/CellViewController.swift (hunk 7 above)
Comment (NikolaBlagoevski) [reply test1/CellViewController.swift]: changed the txt.count part with txt.isEmpty on all occurrences.

**Review by martinwolfware — APPROVED** [review]

File: test1/CellViewController.swift (hunk 5 above)
Comment (martinpesevski) [reply test1/CellViewController.swift:44]: I meant to move the function body higher up in the code. won't block you on this

**Review by martinpesevski — APPROVED** [review]

File: test1/CellViewController.swift (hunk 5 above)
Comment (NikolaBlagoevski) [reply test1/CellViewController.swift:44]: Ok, i will merge then

**Pull request merged**